mlflow/fastai.py

```diff
@@ -77,6 +79,25 @@ def save_model(fastai_learner, path, conda_env=None, mlflow_model=Model(), **kwa
                             ]
                         }
     :param mlflow_model: MLflow model config this flavor is being added to.
+
+    :param signature: (Experimental) :py:class:`ModelSignature <mlflow.models.ModelSignature>`
+                      describes model input and output :py:class:`Schema <mlflow.types.Schema>`.
+                      The model signature can be :py:func:`inferred <mlflow.models.infer_signature>`
+                      from datasets with valid model input (e.g. the training dataset) and valid
+                      model output (e.g. model predictions generated on the training dataset),
+                      for example:
+
+                      .. code-block:: python
+
+                        from mlflow.models.signature import infer_signature
+                        train = df.drop_column("target_label")
+                        signature = infer_signature(train, model.predict(train))
```

Comment:
Not sure that this example will make sense to fastai users (where there's no `model.predict(pandas_df)` API) - maybe we could make it `fastai` specific (preferable but more work) or rewrite it to be generic to ML frameworks (i.e. write something like):

```
                        from mlflow.models.signature import infer_signature
                        train_pandas_df = train_pandas_df.drop_column("target_label")
                        output_pandas_df = ... # compute model predictions & convert to a pandas dataframe
                        signature = infer_signature(input_pandas_df, output_pandas_df)
```

I do think per-framework usage examples would be ideal (make it easiest for users of the framework to actually pass a signature/example), but could come in a follow-up PR.

Reply:
I agree that per framework example would be the best but I don't think it's super important. 
Showing more generic way of generating model predictions makes sense.